presto/utils.py

```diff
@@ -275,3 +276,16 @@ def plot_for_group(grp_df):
     )
 
     metrics_df.groupby(["model", "metric_type"]).apply(plot_for_group)
+
+
+def plot_spatial(spatial_preds: xr.Dataset, output_path: Path, to_wandb: int = False):
```

Comment:
All plots in [this comment](https://github.com/WorldCereal/presto-worldcereal/pull/14#issuecomment-1881287660) were made using this function.